### PR TITLE
Skip createScope on catch without argument

### LINKED ISSUE #20: Try catch without an argument causes an error
I'm getting the following error: `Cannot read property 'name' of null`

when I use the following syntax:

```js
try {
  ...
} catch {
  ...
}
```

If I change it to `catch (e) {` then everything seems to be working. 

Full stack trace:

```
at Object.createScopes [as pre] (/home/eliya/projects/medflyt_webapp/node_modules/ng-annotate-patched/src/scopetools.js:111:24)
at c (/home/eliya/projects/medflyt_webapp/node_modules/ng-annotate-patched/src/traverse.js:11:43)
at Object.base.TryStatement (/home/eliya/projects/medflyt_webapp/node_modules/acorn-walk/dist/walk.js:246:25)
at c (/home/eliya/projects/medflyt_webapp/node_modules/ng-annotate-patched/src/traverse.js:13:41)
at Object.skipThrough (/home/eliya/projects/medflyt_webapp/node_modules/acorn-walk/dist/walk.js:186:39)
at c (/home/eliya/projects/medflyt_webapp/node_modules/ng-annotate-patched/src/traverse.js:13:41)
```

### PR DESCRIPTION
**Pull request by Newbie012**
Fixes https://github.com/bluetech/ng-annotate-patched/issues/20

I'm not sure if it's the right solution, but it seems to work for my end.

### CODE REVIEW

File: src/scopetools.js [hunk 1]
```diff
@@ -101,6 +101,10 @@ function createScopes(node, parent) {
 
     } else if (node.type === "CatchClause") {
         const identifier = node.param;
+        
+        if (identifier === null) {
```
Comment (bluetech) [inline src/scopetools.js:111]: The `node.$scope = new Scope(...)` part should still happen. I suggest we make the code look like this:

```js
else if (node.type === "CatchClause") {
    node.$scope = new Scope({
        kind: "catch-block",
        node: node,
        parent: parent.$scope,
    });

    const identifier = node.param;
    if (identifier !== null) {
        // Rest of the code here
    }
}
```

**Review by bluetech — CHANGES_REQUESTED** [review]
Thanks! I left a comment. Also, a test would be nice - minimally that would be adding a line to `tests/original.js` and update `tests/with_annotations.js` accordingly. You can run the tests with `npm test`.

**Comment by Newbie012** [comment]
That would make sense. I have updated the changes according to your requirements.

**Review by bluetech — APPROVED** [review]
Thanks!

**Comment by bluetech** [comment]
I published version 1.14.0 with this change.

**Comment by Newbie012** [comment]
Thank you!

**Pull request merged**